Week-1/cattyBoard/index.css

```diff
@@ -0,0 +1,106 @@
+.main{
+    display: flex;
+    margin: 0px 0px;
+}
+section{
```

Comment:
for smaller applications, using element name to add css is fine but when we are working on a bigger web page, using element names isnt recommended. Instead use id and class (class is more recommended).